### ISSUE TITLE
Core Developer: My teams work component

### ISSUE THREAD

**Issue by ineedsubstance** [punchcard-cms/punchcard#539]

**Comment by ineedsubstance** [comment]
Blocked by #538

**Comment by ineedsubstance** [comment]
Related to #538 and uses the same component

**Comment by mkiang3051** [comment]
When we talk about `identifier`, does that correspond to the content's `title`? Or is it referencing its `content type`? I think it may be helpful to show both the user's content `title` as well as the `content type`. 
Also, would we want to surface what `revision` number the content is on at this level? Or keep it for the more detailed view? (I only ask b/c the demo shows revisions - but whether it's actually helpful to surface at this level may depend on the users of the CMS)
I'm open to talking more about this later on or when we get to wanting to modify this component.
I just wanted to put my thoughts here before I forget (and before I lose track of this issue!)

**Comment by ineedsubstance** [comment]
Yes, `identifier` is in reference to the title of that content. `Content type` could be useful.

I'm not sure `revision number` is something the user would understand or would help them, but I could be wrong. How would you see that being useful for the user?

**Comment by mkiang3051** [comment]
@ineedsubstance Hmm I'm not sure `revision number` would be needed - but maybe to see if a content already has been worked on/revised over several iterations? We could probably ask Allen or Jeremy to get their thoughts too - since I don't really know how helpful it is to display at a high level.

**Comment by ineedsubstance** [comment]
Yeah, I would be curious. I just don't know how relevant that is, but it's worth asking.

**Comment by mkiang3051** [comment]
Ok cool - I can ping them an ask. Either way, I don't think it'd be a hard thing to change if we wanted to add in `revision` later.
